packages/router/src/RouterLink.ts

```diff
@@ -269,7 +269,13 @@ export interface _RouterLinkI {
       RouterLinkProps
 
     $slots: {
-      default: (arg: UnwrapRef<ReturnType<typeof useLink>>) => VNode[]
+      default?: ({
+        route,
```

Comment:
I'm curious: does the destructuring help with something?
As in practice, this means we need to add a property if `useLink()` changes. If that's the case, can you add a `// NOTE: update `$slots` below` to the return statement of `useLink()`?

Reply:
<img width="668" alt="image" src="https://user-images.githubusercontent.com/8111351/188087649-c4b5276d-62d5-432e-8888-333ea827f2a9.png">

It will show all field of this interface

Reply:
so, better slightly better DX, got it 👍